From 9055eafea7d1df7f26b702bf050e431bb75fd5f7 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Tue, 18 Jul 2023 19:43:02 -0700
Subject: [PATCH] Don't build manpages

---
 autogen.sh | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index 6115a87..b1defea 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -1,6 +1,3 @@
 #!/bin/sh
 
 autoreconf -f -i -s
-
-cd man
-./make.sh
-- 
2.34.1

